tools/v8-prof/tick-processor.js

```
@@ -0,0 +1,39 @@
+'use strict';
+var cp = require('child_process');
+var fs = require('fs');
+var path = require('path');
+
+var toolsPath = path.join(__dirname, '..', '..', 'deps', 'v8', 'tools');
+var scriptFiles = [
+    path.join(__dirname, 'polyfill.js'),
+    path.join(toolsPath, 'splaytree.js'),
+    path.join(toolsPath, 'codemap.js'),
+    path.join(toolsPath, 'csvparser.js'),
+    path.join(toolsPath, 'consarray.js'),
+    path.join(toolsPath, 'csvparser.js'),
+    path.join(toolsPath, 'consarray.js'),
+    path.join(toolsPath, 'profile.js'),
+    path.join(toolsPath, 'profile_view.js'),
+    path.join(toolsPath, 'logreader.js'),
+    path.join(toolsPath, 'tickprocessor.js'),
+    path.join(toolsPath, 'SourceMap.js'),
+    path.join(toolsPath, 'tickprocessor-driver.js')];
+
+var tempScript = path.join(__dirname, 'tick-processor-tmp-' + process.pid);
+var cat = cp.spawn('cat', scriptFiles);
+cat.stdout.pipe(fs.createWriteStream(tempScript));
+cat.stdout.on('end', function() {
```

If there is am unfortunate error, this will fail silently.